events/amta2022.md

```diff
@@ -34,6 +34,19 @@ Students interested in MT and computational linguistics will be able to connect
 
 ## Workshops
 
+The First Workshop on Corpus Generation and Corpus Augmentation for Machine Translation (CoCo4MT) will be co-located with AMTA 2022 in Orlando, Florida, USA, on September 16th, 2022. 
+
+It is the first workshop centered around research focusing on corpora creation, cleansing, and augmentation techniques specifically for machine translation. 
+
+Topics of the workshop include but are not limited to:
+- Difficulties with using existing corpora (e.g., political considerations or domain limitations) and their effects on final MT systems,
+- Strategies for collecting new MT datasets (e.g., via crowdsourcing),
+- Data augmentation techniques,
+- Data cleansing and denoising techniques,
+- Quality control strategies for MT data,
+- Exploration of datasets for pretraining or auxiliary tasks for training MT systems.
+
+More information is available at: 
```

Comment:
To follow the structure in other sections, let's just delete the introductory sentence and just have the link. What do you think?

Reply:
Agree, thank you.